.gitignore

```diff
@@ -23,3 +23,6 @@ local.properties
 .gradle/
 build/
 .externalNativeBuild
+
+# npm
+package-lock.json
```

Comment:
package-lock.json should not be ignored.

The lock file is not in the project just because the packages migrate from the internal repository to the public, the previous lock file will make the installation process got an error.

Reply:
@xuqingkuang done.